vivisect/analysis/i386/calling.py

```diff
@@ -113,13 +113,12 @@ def analyzeFunction(vw, fva):
     cc = emu.getCallingConvention(callconv)
     stcount = cc.getNumStackArgs(emu, argc)
     stackidx = argc - stcount
+    baseoff = cc.getStackArgOffset(emu, argc)
```

Comment:
Are there any calling conventions where this couldn't be a statically declared class-local value?  ( or we could probably make the default impl just return a declared class-local of 0 which things like msfastcall can declare as 12 or whatever? )